docs/NEWS

```diff
@@ -1,3 +1,44 @@
+Avahi 0.9, the 'Odrodzenie' release
+===============================
+
+The avahi 0.9 release is mostly bug fix release, including a set of vulnerabilities.
+No changes to API happened, which would be incompatible.
+
+New Features:
+- Temporary files are stored in runstatedir, /run by default #137
```

Comment:
It isn't `/run` by default. On systemd-based system it should be set to `/run` by analogy with what the Arch package did: https://gitlab.archlinux.org/archlinux/packaging/packages/avahi/-/commit/daafee2d7f703a7f871f709d7f430dc8f1c7f336. On non-systemd based systems it should point to either `/var/run` (https://github.com/avahi/avahi/issues/150) or `/run` depending on where they keep that stuff. The idea is that now it's configurable via the standard variable instead of relying on bespoke things.

I think in terms of packaging it should also be mentioned that avahi depends on `libsystemd` now and that the dependency can safely be turned off on systems with no systemd. (It should probably be added to https://github.com/avahi/avahi/blob/master/docs/README as well).

Reply:
Okay, commit for libsystemd is marked by issues #287, #296. And right, on configure.ac, line 1024, runstatedir defaults to ${localstatedir}/run in avahi, unless provided by autoconf. Which defaults to the same in autoconf-2.71-10.fc40.noarch.